staging/src/k8s.io/apiserver/plugin/pkg/audit/truncate/truncate.go

```diff
@@ -72,10 +72,12 @@ func NewBackend(delegateBackend audit.Backend, config Config, groupVersion schem
 }
 
 func (b *backend) ProcessEvents(events ...*auditinternal.Event) bool {
-	var errors []error
-	var impacted []*auditinternal.Event
-	var batch []*auditinternal.Event
-	var batchSize int64
+	var (
```

Comment:
I've never seen this done inside a method before vs at the package scope.

This is actually more lines and doesn't seem any more readable, seems like pointless bikeshed?